src/components/PatientLine.tsx

```diff
@@ -1,49 +1,106 @@
+import { deletePatient } from '../services/api'
+import { ErrorContext } from '../context/error'
 interface PatientLineProps {
   patient: Patient
   borderColor?: string
 }
 
 const PatientLine: FunctionComponent<PatientLineProps> = ({ patient }) => {
   const navigate = useNavigate()
+  const [buttonsModalShow, setButtonsModalShow] = useState(false)
+  const handleCloseButtonsModal = (): void => setButtonsModalShow(false)
+  const handleShowButtonsModal = (): void => setButtonsModalShow(true)
+  const { t } = useTranslationHook()
+  const { token } = useContext(TokenContext)
 
   const prescription = getValidOrLastPrescription(patient.prescriptions)
   const prescriptionEndDate = prescription?.end_date ?? 'N/A'
   const prescriptionIsValid = prescription?.is_valid ?? false
-  const isValidIconName = prescriptionIsValid ? 'circle-check' : 'circle-xmark'
-  const isValidIconClass = prescriptionIsValid ? 'text-success' : 'text-danger'
-  const icon = (
-    <FontAwesomeIcon
-      icon={['fas', isValidIconName]}
-      className={isValidIconClass}
-    />
-  )
+  const isValidIconClass = prescriptionIsValid ? 'success' : 'danger'
+  const [show, setShow] = useState(false)
+  const { addError } = useContext(ErrorContext)
+  const handleClose = (): void => setShow(false)
+  const handleShow = (): void => setShow(true)
+
+  const goToPatient = (): void => {
+    navigate(`/patients/${patient.id}`)
+    handleCloseButtonsModal()
+  }
+  const handleEdit = async (): Promise<void> => {
+    navigate(`/patients/edit/${patient.id}`)
+    handleCloseButtonsModal()
+  }
 
-  const goToPatient = (id: number): void => {
-    navigate(`/patients/${id}`)
+  const onDelete = async (): Promise<void> => {
+    assert(token)
+    try {
+      await deletePatient(token, patient.id)
+      navigate('/patients')
+    } catch (error) {
+      console.error(error)
+      addError({ body: 'Error deleting patient' })
+    }
+    handleClose()
   }
 
   return (
-    <Card onClick={() => goToPatient(patient.id)} style={{ cursor: 'pointer' }} className='mt-2 shadow'>
-      <Card.Body>
-        <Card.Title>{patient.lastname}</Card.Title>
-        <Card.Subtitle className='mb-2 text-muted'><small>{patient.address} à {patient.city}</small></Card.Subtitle>
-        <Card.Text>
-          <>Renouvellement le: {prescriptionEndDate} - {icon}</>
-          {patient.prescriptions.map((prescription) => (
-            <small key={prescription.id}>
-              Dr. {prescription.prescribing_doctor}
-            </small>
-          ))}
-        </Card.Text>
-      </Card.Body>
-    </Card>
+    <>
+      <Card className='shadow mt-4'>
+        <Card.Body>
+          <div className='row align-items-start'>
+            <div className='col'>
+              <div className='d-flex align-items-center'>
+                <Badge bg='secondary' text='dark' className='me-2'>
+                  {patient.lastname} {patient.firstname}
+                </Badge>
```

Comment:
Why making the patient name so small on a badge? I think this is something that should standout, not be small

Reply:
Yes I agree I going to change